themes/default/content/blog/announcing-speaker-lineup-pulumiup-2023/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Announcing the Speaker Lineup for PulumiUP 2023"
+date: 2023-05-24
+meta_desc: Attend PulumiUP 2023's tech talks and hands-on demos about infrastructure as code, platform engineering, multi-cloud, AI, modern cloud architectures, and more!
+meta_image: meta.png
+authors:
+    - george-huang
+    - sara-huddleston
+    - isaac-harris
+tags:
+    - announcements
+    - pulumi-events
+---
+
+[PulumiUP](https://www.pulumi.com/pulumi-up/) is our annual user conference for the Pulumi community and anyone interested in Infrastructure as Code. Every year, we strive to build a program packed with technical talks, demos, and best practices with the goal of leaving you with new learnings and inspiration that will help you become a better cloud engineer. You'll hear from industry leaders and experts about IaC, software engineering, DevOps, Platform engineering, and AI.
+
+We hope you'll [join us](https://www.pulumi.com/pulumi-up/) and take this opportunity to grow your knowledge and become infrastructure as code stars.
+
+## Get Ready for Amazing Launches and Demos
+
+Learn about Pulumi's newest features across the core IaC SDKs and Pulumi Cloud.
+And watch hands-on demos by the Pulumi team:
+
+* Joe Duffy, Co-founder and CEO
+* Luke Hoban, Co-creator of TypeScript and CTO
+* Meagan Cojocar, Principal Product Manager
+* Evan Boyle, Engineering Manager
+* Zaid Ajaj, Software Engineer
+
+## Final Speakers Announced
+
+You'll hear from innovators, leaders, and experts from multiple industries and learn about infrastructure as code, software engineering, and cloud engineering.
+
+### Sam Cogan - Solution Architect
+
+_Talk: Keeping your Infrastructure Code DRY (Don't Repeat Yourself)_  
+Sam is a Solution Architect with [WTW](https://www.wtwco.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) and a Microsoft Azure MVP. Sam is focused on delivering applications into the cloud using IaaS, PaaS, Serverless, Containers, and Kubernetes. Sam is particularly focused on automation and DevOps, including Infrastructure as Code, Cloud automation tooling, PowerShell, and CI/CD tooling.
```

Comment:
```suggestion
_Talk: Keeping your Infrastructure Code DRY (Don't Repeat Yourself)_  

Sam is a Solution Architect with [WTW](https://www.wtwco.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) and a Microsoft Azure MVP. Sam is focused on delivering applications into the cloud using IaaS, PaaS, Serverless, Containers, and Kubernetes. Sam is particularly focused on automation and DevOps, including Infrastructure as Code, Cloud automation tooling, PowerShell, and CI/CD tooling.
```

Reply:
@SaraDPH We can't add a blank line here because then the files will fail the Markdown linting tests (it will want us to change the italicized lines to headings instead, which isn't quite what we are going for). That will go for all the places you suggested adding a blank line after the italicized "Talk" line.